src/resmom/catch_child.c

```diff
@@ -588,11 +588,11 @@ encode_used(job *pjob, pbs_list_head *phead)
 
 	at = &pjob->ji_wattr[JOB_ATR_resc_used];
 	ad = &job_attr_def[JOB_ATR_resc_used];
-	if ((at->at_flags & ATR_VFLAG_SET) == 0)
+	if ((is_attr_set(at)) == 0)
```

Comment:
remove redundant `()` around `is_attr_set`, here and other places too

Reply:
I know, but this is just not easy to script up, it's there in a ton of places as well so doing it manually is not practical. So I propose that we get rid of them as we see them, logic wise they don't make a difference and they aren't exactly ugly. This will all go away with CPP anyways. But let me know what you think.

Reply:
ok, i actually did fix all the calls to is_attr_set(), but for the other functions it's way too many

Reply:
@agrawalravi90 its fine, we can change as we see them.